source/elements/oneDAL/source/terminology.rst

```diff
@@ -0,0 +1,64 @@
+.. _terminology:
+
+============
+Terminology
+============
+
+@table-begin:
+   column-sizes: [ 25% ]
+Term | Definition \\
+
+Batch Mode | The computation mode for an algorithm in oneDAL, where all the data needed for
+computation are available at the start and fits the memory of the device where computations will
+be performed. \\
+
+Dataset | A pair of a matrix $X$ of size $N \times p$ and a vector $Y$ of size $N$, where $X$
+contains $N$ feature vectors in rows and $Y$ stores all corresponding responses. Vector $Y$ is
+optional and may not be present if responses are unknown. \\
+
+DPC++ | DPC++ --- Technology (programming language, compiler and runtime) that
+is based on SYCL specification and extends it. \\
+
+Feature | A single column of matrix $X$ from a dataset. \\
+Feature vector | A single real-valued vector of length $p$ that encodes information about a real
+object or an event. \\
+
+GEMM | General Matrix Multiplication --- matrix multiplication algorithms in the following form $$C
+= \alpha A^\dagger B^\dagger + \beta C, \; \dagger \in \{1, T\}.$$ \\
+Gen9 | Codename of Intel GPU microarchitecture was used for performance analysis of the algorithms
+in the report. \\
+
+Host/Device | OpenCL terms for referring to CPU that controls the connected GPU executing kernels. \\
+JIT | Just in Time Compilation --- compilation during execution of program. In the report, term is
+used to refer to the compilation of OpenCL(TM) kernels. \\
+
+Kernel | Code written in OpenCL or SYCL and executed on a GPU device. \\
+
+oneMKL | Intel(R) oneAPI Math Kernel Library (Intel(R) oneMKL). \\
+
+Observation | A pair of a feature vector and a response if response is known, otherwise a feature
+vector. \\
+
+Model | A result of training machine-learning algorithm that stores information necessary to run
+inference (prediction) on new dataset. \\
+
+oneDAL | oneAPI Data Analytics Library (oneDAL). \\
+
+Online Mode | The computation mode for an algorithm in oneDAL, where the data needed for computation
+becomes available in parts during the time. \\
+
+Response | A single value corresponding to an observation. For example, response may be a label
+indicating that an observation belongs to a particular class. \\
+
+Skylake | Codename of Intel CPU microarchitecture was used for performance analysis of the
+algorithms in the report. \\
+
+SPIR-V | Language for intermediate representation of compute kernels. \\
+
+SYCL | SYCL(TM) --- high-level programming model for OpenCL(TM) that enables code for heterogeneous
```

Comment:
reference to the standard can be good. Generally, is it okay/recommended to refer to it publicly?

Reply:
I've added reference. Let's keep mention now and later discuss in what places we would like to use it.